cmd/mirror.go

```diff
@@ -223,7 +294,26 @@ func newMirrorPublishCmd() *cobra.Command {
 				return err
 			}
 
-			t := remote.New(endpoint, args[0], args[1], args[3]).WithDesc(desc).WithOS(goos).WithArch(goarch)
+			flagSet := set.NewStringSet()
+			cmd.Flags().Visit(func(f *pflag.Flag) {
+				flagSet.Insert(f.Name)
+			})
+			m, err := env.V1Repository().FetchComponentManifest(args[0])
+			if err != nil {
```

Comment:
Do we need to check the error type, e.g: NotFound?

Reply:
The best way is do so. But it's a little hard to do this in this stage(we didn't define error well yet). So this is a tradeoff.